infra/gcp/terraform/k8s-infra-oci-proxy-prod/oci-proxy.tf

```diff
@@ -96,8 +81,15 @@ resource "google_cloud_run_service" "oci-proxy" {
     spec {
       service_account_name = google_service_account.oci-proxy.email
       containers {
-        image = local.image
-         args = [ "-v=2" ]
```

Comment:
why the verbosity of `archeio` is removed ?

Reply:
fixed